puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/clone_foreman.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -ex
+
+rm -rf foreman/
+git clone https://github.com/theforeman/foreman --branch "${foreman_branch}"
```

Comment:
The `foreman_branch` parameter seems to be undefined: http://ci.theforeman.org/job/test_katello_core/16490/database=postgresql,ruby=1.9.3,slave=fast/console.  Some slaves (EL6?) are using a default branch instead, some aren't.